beta-src/src/components/map/variants/classic/components/WDBoardMap.tsx

```diff
@@ -229,6 +259,16 @@ const WDBoardMap: React.FC = function (): React.ReactElement {
           />
         </pattern>
       </defs>
+      <defs>
+        <pattern
+          id="sea-texture"
+          patternUnits="userSpaceOnUse"
+          width="1546"
+          height="1384"
+        >
+          <image href={WaterTexture} x="0" y="0" width="1966" height="1615" />
+        </pattern>
+      </defs>
```

Comment:
I think you should be able to add this to the existing `defs` container that starts on line 246.